### ISSUE TITLE
(OUI Docs) Breadcrumbs

### ISSUE THREAD

**Issue by KrooshalUX** [opensearch-project/oui#194]
- [x] Update styling to reflect new breadcrumb style implemented in component
- [ ] Identify opportunities to introduce any new material to explain implementation, if any - most implementation details outside of styling should remain the same.

**Comment by BSFishy** [comment]
Reference #131 for tracking of the first point

**Comment by KrooshalUX** [comment]
Blocked until  #131  complete

**Comment by kavilla** [comment]
Should be unblocked with issues created as fast follows

**Comment by KrooshalUX** [comment]
@BSFishy yes, the only changes scoped for the next few months is to reflect any component changes into the OUI docs to make sure we are providing builders the correct experience.

**Comment by anirudha** [comment]
Closing — this documentation issue has been open since 2022/2023 with no progress. The doc site has been significantly updated since then. If specific documentation gaps remain, please open a new issue with current context.
